### ISSUE TITLE
MinMaxScaler() usage

### ISSUE THREAD

**Issue by formidiable** [MehranTaghian/DQN-Trading#5]
**Warning Risk of data leak Do not use [minmax_scale](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.minmax_scale.html#sklearn.preprocessing.minmax_scale) unless you know what you are doing. A common mistake is to apply it to the entire data before splitting into training and test sets. This will bias the model evaluation because information would have leaked from the test set to the training set. In general, we recommend using [MinMaxScaler](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html#sklearn.preprocessing.MinMaxScaler) within a [Pipeline](https://scikit-learn.org/stable/modules/compose.html#pipeline) in order to prevent most risks of data leaking: pipe = make_pipeline(MinMaxScaler(), LogisticRegression()).**

from https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.minmax_scale.html .
